engine/bench_test.go

```diff
@@ -203,6 +203,14 @@ func BenchmarkRangeQuery(b *testing.B) {
 			name:  "histogram_quantile",
 			query: `histogram_quantile(0.9, http_response_seconds_bucket)`,
 		},
+		{
+			name:  "sort",
+			query: `sort(http_response_seconds_bucket)`,
+		},
+		{
+			name:  "sort_desc",
+			query: `sort(http_response_seconds_bucket)`,
```

Comment:
Small nit. Should be `sort_desc`.

Reply:
:heavy_check_mark:

Reply:
Thanks!

Reply:
Though i wonder if that is even a great benchmark since semantics of `sort` for a range query are not clear to me right now

Reply:
Yeah, I think it is not clear at all what's the expected behavior in this case.